Tests/Sentry.Xamarin.Tests/Extensions/IHubExtensionsTests.cs

```diff
@@ -0,0 +1,154 @@
+﻿using Sentry.Extensions;
+using NSubstitute;
+using Xunit;
+using System.Collections.Generic;
+using System;
+using Sentry.Extensibility;
+using System.Threading.Tasks;
+
+namespace Sentry.Xamarin.Tests.Extensions
+{
+    public class IHubExtensionsTests
+    {
+        public IHub Sut { get; set; } = Substitute.For<IHub>();
+        public Scope Scope { get; set; } = new Scope(null);
+
+        public IHubExtensionsTests()
+        {
+            Sut.When(h => h.ConfigureScope(Arg.Any<Action<Scope>>()))
+                .Do(c => c.Arg<Action<Scope>>()(Scope));
+
+        }
+
+        [Fact]
+        public void AddInternalBreadcrumb_LastBreadCrumbNull_LastBreadcrumbSet()
```

Comment:
`breadcrumb` is a word:

```suggestion
        public void AddInternalBreadcrumb_LastBreadcrumbNull_LastBreadcrumbSet()
```